src/main/java/org/jabref/gui/entryeditor/citationrelationtab/semanticscholar/SemanticScholarFetcher.java

```diff
@@ -71,6 +70,14 @@ public List<BibEntry> searchCiting(BibEntry entry) {
         return new ArrayList<>();
     }
 
+    public static RelatedEntriesFetcher buildCitationsFetcher() {
+        return entry -> new SemanticScholarFetcher().searchCitedBy(entry);
```

Comment:
On each entry, a new `SemanticScholarFetcher` instance is generated. I think, an instance of fetcher can be called multiple times. Thus, create the `SemanticScholarFetcher` and then use that in the lambda.

Reply:
Done.